File: EMPLOYABILITY SKILL PROGRAM/Week 1/Question 1/Solution.cpp
#include <iostream>
using namespace std;

int main() {
    int num1, num2;

    cout << "Enter the first number:";
    cin >> num1;

    cout << "Enter the Second number:";
    cin >> num2;

    if (num1 % num2 == 0)
        cout << "First number", num1, "is multiple of second number";
    else
        cout << "First number", num1, "is not multiple of second number";

    return 0;
}

File: EMPLOYABILITY SKILL PROGRAM/Week 1/Question 1/Solution.py
#Write a program to enter two numbers and check whether the first one is multiple of second number or not.

number1 = int(input("Enter first number: "))
number2 = int(input("Enter Second number: "))

if (number1 % number2 == 0):
    print(number1, "is multiple of", number2)
else:
    print(number1, "is not a multiple of", number2)